### DISCUSSION TITLE
✅ Using search form method POST instead of GET

### DISCUSSION THREAD

**Discussion by ZenboxMarketing** [elementor/elementor#21179] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

To achieve better security, I'd like to change the method of the search feature from "get" to "post". I'm using the search form element in the website's navigation. 

### Describe the solution you'd like

An ideal solution would be to change the method in the search form element settings. I would also be happy with code that I can add to the child theme's functions.php

### Additional context

![get-method](https://user-images.githubusercontent.com/123819577/215272533-a3b17a12-c579-468c-b940-a4be78a644d1.jpg)

**Comment by nicholaszein** [comment]
Hi @ZenboxMarketing!

We have great news! 🙌 

### 📢 We're happy to announce that we released a new Search widget with Elementor Pro v3.23! 🥳 

You can now use the [new Search widget with Live Results](https://elementor.com/help/search-widget/) without having to worry about security, as the request uses AJAX with a nonce, and communicates internally via REST-API, without needing to expose any endpoints.

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂

**Comment by ZenboxMarketing** [comment]
@nicholaszein thank you so much for responding with this info!
